### PR TITLE
docs(readme): provide initial README

### PR DESCRIPTION
**Pull request by skadisch**

### CODE REVIEW

**Comment by peterbornerup** [comment]
Updated title - don't think this should trigger a new version :)

**Review by peterbornerup — APPROVED** [review]
Awesome :D

File: README.md [hunk 1]
```diff
@@ -1 +1,163 @@
-# node-lksctp
\ No newline at end of file
+# node-lksctp
+
+## System requirements
+
+Requires libsctp of [LKSCTP](https://github.com/sctp/lksctp-tools), also known as libsctp-dev debian package.
+
+It has been tested to work with v1.0.19 and v1.0.20, but most likely also supports older versions and newer versions.
```
Comment (latysheff) [inline README.md:7]: It is known to fail on v1.0.18 :)

Comment (skadisch) [reply README.md:7]: As discussed, should be supported if compile time has right environment

**Pull request merged**